### PR TITLE
Setup docker

### LINKED ISSUE #10: SHOULD: Datenbank Docker Container
Als Entwickler/User möchte ich die Datenbank mittels einem Docker Container starten können, um das Datenbank Setup zu überbrücken.

### PR DESCRIPTION
**Pull request by ladral**
Actually the docker compose contains just the database container. Additional containers can be added later.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: "3.3"
+
+services:
+  db:
+    image: mariadb:latest
```
Comment (Hubbacca) [inline docker-compose.yml]: latest!=aktuell. Fixe Version verwenden.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+[![Build Status](https://travis-ci.com/ibwgr/warranty-tracker.svg?branch=master)](https://travis-ci.com/ibwgr/warranty-tracker) <br/>
+develop <br/>
+[![Build Status](https://travis-ci.com/ibwgr/warranty-tracker.svg?branch=develop)](https://travis-ci.com/ibwgr/warranty-tracker) <br/>
+
+## Installation
+### MariaDB
+Set up mariaDB for local development and testing using Docker: 
+1. install docker on your host
+2. copy the dev.env file and save it as .env
+3. fire up your terminal and go to the root directory of the project
```
Comment (Hubbacca) [inline README.md]: works like a charm, but that docker must be started before running the command 'docker-compose up' should also be stated.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  db:
+    image: mariadb:10.5
+    # Variables values have to be set from outside, e.g. via .env file
+    environment:
+      - MYSQL_ROOT_PASSWORD
+      - MYSQL_DATABASE
+    volumes:
+      - ./database:/docker-entrypoint-initdb.d
+    ports:
+      - 3333:3306
```
Comment (Hubbacca) [inline docker-compose.yml:13]: Theoretically can Port 3333 be replaced with 3306 (mariadb standard port). Maybe an addition in the installation guide like: "Attention, Port 3306 (or 3333 whatever is taken) must not be occupied"

**Review by Hubbacca — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (ladral) [reply docker-compose.yml:13]: The readme file contains now the information about the port

File: README.md (hunk 2 above)
Comment (ladral) [reply README.md]: updated the instruction in the readme file.

File: docker-compose.yml (hunk 1 above)
Comment (ladral) [reply docker-compose.yml]: changed to version 10.5

**Pull request merged**